test/integration/cluster/test_cluster_publish.py

```diff
@@ -624,16 +627,44 @@ def test_cluster_publish_config_etcd_key_not_exist(tt_cmd, tmpdir_with_cfg):
     assert expected in publish_output
 
 
-def test_cluster_publish_config_etcd_no_auth(tt_cmd, tmpdir_with_cfg, etcd):
+@pytest.mark.parametrize(
+    "instance_name, expected_err_msg",
+    [
+        pytest.param(
+            "etcd",
+            r"   ⨯ failed to fetch data from etcd: etcdserver: user name is empty",
+        ),
+        pytest.param(
+            "tcs",
+            r"   ⨯ failed to put data into tarantool: Execute access to"
+            " function 'config.storage.txn' is denied for user",
+        ),
+    ],
+)
+def test_cluster_publish_config_no_auth(
+    instance_name, tt_cmd, tmpdir_with_cfg, request, expected_err_msg
+):
+    # Skip test if tarantool is not EE edition and if its major version less than 3.
+    if instance_name == "tcs":
+        if tarantool_major_version < 3 or not utils.is_tarantool_enterprise():
```

Comment:
It could a useful util method too.
```suggestion
        if utils.is_tarantool_less_3() or not utils.is_tarantool_enterprise():
```

Reply:
Added this method.